kahuna/public/js/search/query.js

```diff
@@ -228,6 +252,13 @@ query.controller('SearchQueryCtrl', [
         } else {
           ctrl.filter.nonFree = undefined;
         }
+
+        if (orgOwned === null) {
+            storage.setJs("orgOwned",ctrl.filter.orgOwned);
```

Comment:
Does `ctrl.filter.orgOwned` need some additional checks here, similar to `ctrl.filter.nonFree` above?